locale/en.po

```diff
@@ -2275,10 +2275,14 @@ msgstr "HP:"
 msgid "HSV"
 msgstr "HSV"
 
+#: ../src/microbe_stage/gui/HUDMessages.cs:37
+msgid "HUD_MESSAGE_MULTIPLE"
+msgstr "{0} (x{1})"
```

Comment:
Should `SPECIES_N_TIMES` be replaced with this?

Reply:
I think the usage context is quite different, which is why I did not combine these. For example in Finnish if the "x2" wasn't understandable, different substitute words would make sense to use in these two different contexts.